### ISSUE TITLE
support subscriptions on remote schemas

### ISSUE THREAD

**Issue by dsandip** [hasura/graphql-engine#1599]

**Comment by EddieVallejos** [comment]
+1

**Comment by bertlokeen** [comment]
+1

**Comment by dasimandl** [comment]
@ecthiender looking forward to this enhancement. What is the updated timeline for the release of this feature?

**Comment by Tbaut** [comment]
we're very much looking forward this feature, once the (remote schema feature is in). Easy to setup for a subscriptions server it the essence or Hasura :rocket:

**Comment by MaxmaxmaximusGitHub** [comment]
I completely agree, we have a need for these architectures, and now we can not implement them through one endpoint

![image](https://user-images.githubusercontent.com/60403187/82670829-8880af00-9c46-11ea-87c2-d603974c6017.png)

![image](https://user-images.githubusercontent.com/60403187/82670839-8caccc80-9c46-11ea-8d10-0e7f3b13c94c.png)

**Comment by sarendsen** [comment]
+1

**Comment by ironsalsa** [comment]
+1 This would be a killer feature on top of the already killer Subscriptions feature! Even if it's similar to the poll-every-second way that it works for Postgres data.

**Comment by joni7777** [comment]
+1 really need this feature 👍

**Comment by jflambert** [comment]
+1 would love remote subscriptions

**Comment by naderfares** [comment]
This is the only feature we are waiting to adopt Hasura.

**Comment by chazmuzz** [comment]
This is a real pain point for us right now

**Comment by BenediktMiller** [comment]
Support for this would be great!

**Comment by L-U-C-K-Y** [comment]
Is there any update on this part, something that you would consider in the near future?

**Comment by cristianounix** [comment]
+1

**Comment by rahulagarwal13** [comment]
Thank you for the request and comments for this feature. We would like to inform you that this is on our roadmap but we do not have a timeline at present. Please continue to follow this Github issue. We plan to publish on this issue a detailed RFC that covers all use cases and limitations of the feature. We welcome more detailed feedback from you once we provide those details.

For now it would be useful to understand more about your application/API design that requires you to use a remote schema subscription. Also we recently launched [streaming subscriptions](https://hasura.io/docs/latest/subscriptions/postgres/streaming/index/) along with already supported functionality of [live queries](https://hasura.io/docs/latest/subscriptions/postgres/livequery/index/). We would be interested to know if your remote schema use-case is more streaming or live-query related.

**Comment by jflambert** [comment]
we use streaming and regular subscriptions equally, depending if our dataset is timeseries or a dimension respectively.

having said that, since this issue has been open for a long time, we converted all our remote schema queries to native postgres data sources.

our real killer need now is to allow subscriptions on remote relationships.

**Comment by seanparkross** [comment]
Docs has a PR in place to indicate that Subscriptions are not supported for the time being on Remote Schemas.

**Comment by humb1t** [comment]
+1

**Comment by PanchoBubble** [comment]
+1

**Comment by geoffraymond** [comment]
Any updates on this?

**Comment by evansendra** [comment]
+1

**Comment by robcurtis** [comment]
Will this be supported in Hasura V3?

**Comment by NguyenAnhVuong** [comment]
+1

**Comment by seanparkross** [comment]
It depends on the connector. For the Postgres v3 connector it's at the RFC stage now. 

@robcurtis @NguyenAnhVuong

**Comment by talesporto** [comment]
+1
